data/apply-updates-non-planet-tables.sql

```diff
@@ -1,3 +1,23 @@
+-- correct the invalid data in the ne tables from reprojection
+UPDATE ne_10m_lakes SET the_geom=ST_MakeValid(the_geom) WHERE NOT ST_IsValid(the_geom);
+UPDATE ne_110m_land SET the_geom=ST_MakeValid(the_geom) WHERE NOT ST_IsValid(the_geom);
+
+UPDATE ne_10m_admin_0_boundary_lines_land
+  SET
+    the_geom = (CASE WHEN GeometryType(ST_MakeValid(the_geom)) = 'MULTILINESTRING'
+                          THEN ST_MakeValid(the_geom)
+                     ELSE the_geom END)
+  WHERE NOT ST_IsValid(the_geom);
+DELETE FROM ne_10m_admin_0_boundary_lines_land WHERE NOT ST_IsValid(the_geom);
+
+UPDATE ne_10m_admin_1_states_provinces_lines
+  SET
+    the_geom = (CASE WHEN GeometryType(ST_MakeValid(the_geom)) = 'MULTILINESTRING'
+                          THEN ST_MakeValid(the_geom)
+                     ELSE the_geom END)
```

Comment:
In fact, I think most of them fall into this category, and that's why I didn't leave it as a simple update. I drop them on the next line. Is there a better alternative?